### PR TITLE
fix tx list page

### PR DESCRIPTION
**Pull request by syuhei176**
fix fetch
modify tx list to show amount of state

### CODE REVIEW

File: src/components/TransactionList.js [hunk 1]
```diff
@@ -2,34 +2,40 @@ import Link from 'next/link'
...
             href={`/transaction?blockNumber=${transaction.blockNumber}&depositContractAddress=${transaction.depositContractAddress}&start=${transaction.range.start}&end=${transaction.range.end}`}
             key={transaction.hash}
           >
             <div className="tx">
-              <div className="hash">{transaction.hash}</div>
-              <div className="time-stamp">{transaction.timestamp}</div>
-              <div className="sender">{transaction.from}</div>
-              <div className="state-object-address">
-                {transaction.stateObject.address}
+              <div className="hash">{i}</div>
```
Comment (daiki44) [inline src/components/TransactionList.js:32]: I guess it's better to change the class name to "index".

File: src/pages/transactions.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ export default function Transactions() {
   const {
     query: { blockNumber }
   } = useRouter()
+  console.log('aaa', blockNumber)
```
Comment (daiki44) [inline src/pages/transactions.js:9]: console.log

File: src/components/TransactionList.js (hunk 1 above)
Comment (syuhei176) [reply src/components/TransactionList.js]: fixed

File: src/pages/transactions.js (hunk 2 above)
Comment (syuhei176) [reply src/pages/transactions.js]: fixed now

File: src/components/TransactionList.js [hunk 3]
```diff
@@ -2,34 +2,40 @@ import Link from 'next/link'
...
             href={`/transaction?blockNumber=${transaction.blockNumber}&depositContractAddress=${transaction.depositContractAddress}&start=${transaction.range.start}&end=${transaction.range.end}`}
             key={transaction.hash}
           >
             <div className="tx">
-              <div className="hash">{transaction.hash}</div>
-              <div className="time-stamp">{transaction.timestamp}</div>
-              <div className="sender">{transaction.from}</div>
-              <div className="state-object-address">
-                {transaction.stateObject.address}
+              <div className="index">{i}</div>
```
Comment (daiki44) [inline src/components/TransactionList.js:32]: I think the first number seems to be 1!
スクリーンショット 2020-08-09 13 30 09

**Pull request merged**